Makefile

```diff
@@ -25,13 +25,11 @@ install-with-mruby:
 	go install -tags mrb ./...
 
 build:
-	env go build -tags mrb -ldflags $(LD_FLAGS) -o $(OUTPUT) cmd/anycable-go/main.go
-
-prepare-cross-mruby:
-	(cd vendor/github.com/mitchellh/go-mruby && MRUBY_CROSS_OS=linux MRUBY_CONFIG=../../../../../../etc/build_config.rb make libmruby.a)
+	go build -tags mrb -ldflags $(LD_FLAGS) -o $(OUTPUT) cmd/anycable-go/main.go
 
 prepare-mruby:
-	(cd vendor/github.com/mitchellh/go-mruby && MRUBY_CONFIG=../../../../../../etc/build_config.rb make libmruby.a)
+	cd vendor/github.com/mitchellh/go-mruby && \
```

Comment:
**Shell Scripting 201**

Just to explain why it was changed: less confusion and more readability. 
Why can we do this: each make command is performed in a separate shell session
Why can't we write it with just separate lines: each line in command is performed in a separate shell session

Outside of a makefile, e.g. on CI, `pushd` and `popd` are usually used.